src/core/doall/src/DOALLTask.cpp

```diff
@@ -45,13 +45,17 @@ void DOALLTask::initializeTask(Function *F) {
   this->taskInstanceID = (Value *)&*(argIter++);
   this->numTaskInstances = (Value *)&*(argIter++);
   this->chunkSizeArg = (Value *)&*(argIter++);
+  this->scylaxPerThreadDataArg = (Value *)&*(argIter++);
```

Comment:
Can we come up with a more descriptive name for this field?

Reply:
It depends if Scylax is an acceptable name to describe what GINO does for I/O at runtime. I'm not committed to it having a specific name at all but it has been useful to lexically group things.

Reply:
Per IRL: Add comment about Scylax name near struct defs